### PR TITLE
ci: fix install lite on GPUs

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Seems we were installing PL instead of Lite :otter: 

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @carmocca @akihironitta @borda

### CODE REVIEW

File: .azure/gpu-tests-lite.yml [hunk 1]
```diff
@@ -62,18 +62,19 @@ jobs:
...
-        python ./requirements/pytorch/adjust-versions.py requirements/lite/base.txt ${PYTORCH_VERSION}
-
         CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
         pip install -e .[dev,strategies] --find-links https://download.pytorch.org/whl/cu${CUDA_VERSION_MM}/torch_stable.html
+      displayName: 'Adjust dependencies'
 
+    - bash: |
+        set -e
+        CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
+        pip install -e .[dev,strategies] --find-links https://download.pytorch.org/whl/cu${CUDA_VERSION_MM}/torch_stable.html
```
Comment (carmocca) [inline .azure/gpu-tests-lite.yml:72]: I don't understand the fix. Why are we doing pip install twice now?

File: .azure/gpu-tests-lite.yml [hunk 2]
```diff
@@ -62,18 +62,19 @@ jobs:
       displayName: 'set visible devices'
 
     - bash: |
-        set -e
-        PYTORCH_VERSION=$(python -c "import torch; print(torch.__version__.split('+')[0])")
-        python ./requirements/pytorch/adjust-versions.py requirements/lite/base.txt ${PYTORCH_VERSION}
-
         CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
         pip install -e .[dev,strategies] --find-links https://download.pytorch.org/whl/cu${CUDA_VERSION_MM}/torch_stable.html
```
Comment (akihironitta) [inline .azure/gpu-tests-lite.yml:66]: I think these two lines can be completely removed because they're duplicates of the following step
```suggestion
        CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
        pip install -e .[dev,strategies] --find-links https://download.pytorch.org/whl/cu${CUDA_VERSION_MM}/torch_stable.html
```

File: .azure/gpu-tests-lite.yml [hunk 3]
```diff
@@ -62,18 +62,19 @@ jobs:
       displayName: 'set visible devices'
 
     - bash: |
-        set -e
-        PYTORCH_VERSION=$(python -c "import torch; print(torch.__version__.split('+')[0])")
-        python ./requirements/pytorch/adjust-versions.py requirements/lite/base.txt ${PYTORCH_VERSION}
```
Comment (akihironitta) [inline .azure/gpu-tests-lite.yml:67]: We need to keep `adjust-versions.py` to adjust the PyTorch version, no?

**Review by akihironitta — APPROVED** [review]

File: .azure/gpu-tests-lite.yml (hunk 3 above)
Comment (carmocca) [reply .azure/gpu-tests-lite.yml:67]: I have the same question

Comment (Borda) [reply .azure/gpu-tests-lite.yml:67]: ok, I misunderstood you question :)

**Review by otaj — APPROVED** [review]

**Review by akihironitta — APPROVED** [review]
